politeiawww/api/www/v1/v1.go

```diff
@@ -56,6 +56,7 @@ const (
 	RouteAllVoteStatus            = "/proposals/votestatus"
 	RouteProposalPaywallDetails   = "/proposals/paywall"
 	RouteProposalPaywallPayment   = "/proposals/paywallpayment"
+	RouteShortProposalDetails     = "/proposals/{tokenPrefix:[A-z0-9]{6}}"
```

Comment:
Would prefer the name `RouteProposalDetailsShort` so that the naming lines up, looks nice, and is easier to read.

```
RouteProposalDetails
RouteProposalDetailsShort

handleProposalDetails
handleProposalDetailsShort

etc...
```

Reply:
Also, there still needs to be a `ProposalDetailsShort` struct defined in `v1.go`.  Same as the user details route.  You don't need to redefine the reply, but make a note of it in the comment (see the Me route as an example).